integration/simulation/network/obscuro_node_utils.go

```diff
@@ -24,7 +26,7 @@ import (
 	"github.com/obscuronet/go-obscuro/integration/simulation/stats"
 )
 
-func startInMemoryObscuroNodes(params *params.SimParams, stats *stats.Stats, genesisJSON []byte, l1Clients []ethadapter.EthClient) ([]rpcclientlib.Client, map[string][]rpcclientlib.Client) {
+func startInMemoryObscuroNodes(params *params.SimParams, stats *stats.Stats, genesisJSON []byte, l1Clients []ethadapter.EthClient) ([]rpcclientlib.Client, map[string][]*obsclient.AuthObsClient) {
```

Comment:
a bit unintuitive for the `startInMemObscNodes` to return the map of `AuthObsClient`.

Shouldn't we be able to build those based on the `rpcclientlib.Client` and each wallet ?

Reply:
Yeah good idea, there's a similar method for the network nodes. It's a refactor of all those methods though, I'll do it in a follow-up PR to avoid confusion.